SECURITY.md

```diff
@@ -0,0 +1,83 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `Gist Extension`
+project.
+
+  * [Reporting a Bug](#reporting-a-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `Gist Extension` team and community take all security bugs in `Gist Extension` seriously.
+Thank you for improving the security of `Gist Extension`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by direct messaging the lead maintainer at @kenhowardpdx.
+
+The lead maintainer will acknowledge your message within 48 hours, and will send a
```

Comment:
I don't want to be beholden to 48 hours. I like the concept of a security policy... but not sure I will actually be able to adhere to this.

Reply:
I think you can merge the PR and do as you wish?  These are the defaults, I'm sure you can tailor them as you like for a personal github project.

https://snyk.io/blog/ten-git-hub-security-best-practices/

Glad I could contribute and appreciate your speedy response to the initial security questions I had and appreciate your work on this extension.